src/mode/mode_command.ts

```diff
@@ -18,22 +22,22 @@ export default class CommandMode extends Mode {
     HandleKeyEvent(key : string) : void {
         this.keyHistory.push(key);
 
+                
         switch (key) {
             case ':':
                 showCmdLine();
                 break;
-            case 'h':
-                vscode.commands.executeCommand("cursorLeft");
-                break;
-            case 'j':
-                vscode.commands.executeCommand("cursorDown");
-                break;
-            case 'k':
-                vscode.commands.executeCommand("cursorUp");
-                break;
-            case 'l':
-                vscode.commands.executeCommand("cursorRight");
-                break;
+            default:
+                var map = this.motionMappings[key];

```

Comment:
I think we need to implement one action soon to see how this fits in the bigger picture. For now it should be fine.